discover/portal/articles-dxp/190-managing-users/11-auditing-users/02-configuring-user-audits.markdown

```diff
@@ -0,0 +1,149 @@
+ # Configuring Audits [](id=configuring-audits)
+
+Audits are enabled by default. As described above, the Control Panel app reports
+audit events, but you can also report them in Liferay's logs or console, enable
+them for scheduled jobs, or disable them entirely. 
+
+There are two main ways to configure @product@:
+
+1.  Edit a configuration via Liferay's Control Panel and the configuration is
+    saved to Liferay's database.
+
+2.  Edit a configuration via an OSGi configuration file (`.config` file) in your
+    `[LIFERAY_HOME]/osgi/modules` folder.
+
+These methods apply to each of the audit configuration options explained below.
+
+## Reporting Audit Events in Liferay's Logs and Console [](id=reporting-audit-events-in-liferays-logs-and-console)
+
+**In the Control Panel:** Go to *Control Panel* &rarr; *Configuration* &rarr;
+*System Settings* and find the *Security* section. Click *Audit* &rarr; *Logging
+Message Audit Message Processor*.
+
+Check *Enabled* to report audit events in Liferay's log. Check *Output to
+Console* if you also want them in the console. Audit events are available in
+either CSV or JSON formats. Select the one you prefer and click *Save*.
+
+To report audit events in Liferay's logs and console via an OSGi configuration
+file, create a file called
+`com.liferay.portal.security.audit.router.configuration.LoggingAuditMessageProcessorConfiguration.config`
+in `[LIFERAY_HOME]/osgi/modules` folder containing these properties:
+
+	enabled="true"
+	logMessageFormat="CSV"
+	#logMessageFormat="JSON"
+	outputToConsole="true"
+
+As you can see, these are the same options from the Control Panel. Edit them
+you see fit.
+
+Regardless of configuration approach, you must also extend Liferay's
```

Comment:
@jhinkey is logging part of core infra? can you read this section and tell me if anything needs to be changed?